src/components/Post/Comments/Comment/Styles.ts

```diff
@@ -27,15 +28,18 @@ export const Dot = styled.div`
   margin: 0 4px;
 `;
 
-export const DropdownMenu = styled(UDropdownMenu)`
-  margin-right: -8px;
-`;
+export const DropdownMenu = styled(UDropdownMenu)``;
 
-export const Middle = styled.div`
+export const CommentSection = styled.div`
+  background: #f4f5f6;
```

Comment:
Instead of hardcoding the color, can we select one from colors? 
`import {colors} from 'styles';`